balancer-js/examples/joinGeneralised.ts

```diff
@@ -146,9 +153,12 @@ async function join() {
   // Use SDK to create join using either Tenderly or VaultModel simulation
   // Note that this does not require authorisation to be defined
   const { expectedOut } = await balancer.pools.generalisedJoin(
-    bbausd2.id,
-    tokensIn,
-    amountsIn,
+    // bbausd2.id,
+    // tokensIn,
+    // amountsIn,
+    '0x02d928e68d8f10c0358566152677db51e1e2dc8c00000000000000000000051e',
```

Comment:
I think it's interesting to use the example while figuring out what's going on, but in order to submit the fix, I believe it's better to revert the example to the previous state and create an integration test to make sure generalisedJoin works by joining with ETH.
Can you please make this change?

Reply:
Yeah! You're totally right, I'll do that